droplet.go

```diff
@@ -402,8 +383,7 @@ func dropletActionPowercycle(ctx *cli.Context) {
 
 func dropletActionShutdown(ctx *cli.Context) {
 	if ctx.Int("id") == 0 && len(ctx.Args()) != 1 {
-		fmt.Printf("Error: Must provide ID or name for Droplet to destroy.\n")
-		os.Exit(1)
+		log.Fatal("Error: Must provide ID or name for Droplet to issue shutdown.")
```

Comment:
some messages are punctuated (dot), some aren't. would be good to have them all punctuated